### ISSUE TITLE
Failed to connect with local Steam Client process.

### ISSUE THREAD

**Issue by Spindermoon1** [Andrey1994/game_overlay_sdk#10]
![image](https://user-images.githubusercontent.com/40631220/118969883-46cc3680-b976-11eb-8f14-eb9a6e555472.png)
This is the command I used to run the program: **python test.py --exe_path "D:\SteamLibrary\steamapps\common\Counter-Strike Global Offensive\csgo.exe" --steam_app_id 730**
I run python as an admin.
Steam is up-to-date.

**Comment by Spindermoon1** [comment]
![image](https://user-images.githubusercontent.com/40631220/118969883-46cc3680-b976-11eb-8f14-eb9a6e555472.png)
This is the command I used to run the program: **python test.py --exe_path "D:\SteamLibrary\steamapps\common\Counter-Strike Global Offensive\csgo.exe" --steam_app_id 730**
I run python as an admin.

**Comment by Andrey1994** [comment]
Hi, it looks like an error from Steam, not from this lib itself....
I dont know why it fails to launch game
Do you see that for all steam games or only for this one? Did you try to update steam client as written here?

**Comment by Spindermoon1** [comment]
Yeah I tried with another game, it launched but no overlay was shown. I had updated steam to the latest version. I see also it has a remote thread error, is that normal?
